website/i18n/en/docusaurus-plugin-content-docs/current/concepts/app-splitting.md

```diff
@@ -18,10 +18,11 @@ The first level of separation: according to the **scope of responsibility** of t
 └── src/
     ├── app/                    # Initializing application logic
     ├── processes/              # (Optional) Application processes running over pages
-    ├── pages/                  # Application Pages
-    ├── features/               # Crucial functionality of the application
-    ├── entities/               # Business entities
-    └── shared/                 # Reused modules
+    ├── pages/                  # Application pages
+    ├── widgets/                # Independent and self-contained blocks of pages
+    ├── features/               # (Optional) Processing of user scenarios
+    ├── entities/               # (Optional) Business entities that domain logic operates with
+    └── shared/                 # Reused modules, non business-specific
```

Comment:
"non business specific"

https://forums.penny-arcade.com/discussion/62699/grammar-issue-quot-non-work-related-quot-where-to-put-hyphens